agir/lib/components/utils/log.js

```diff
@@ -0,0 +1,10 @@
+import debug from "debug";
+
+const generateLogger = (filename) => ({
+  debug: debug(`agir:${filename.replace("/", ":").replace(".js", "")}`),
+  warn: console.warn,
+  info: console.info,
+  error: console.error,
+});
```

Comment:
On aura que trois methodes de `console` ? L'idée est qu'on continue à utiliser console directement pour les autres ? 

Sinon, si on voulait le remplacer partout on pourrait faire une chose comme ça : 

```suggestion
let defaultLogger = {};
if (typeof window !== "undefined") {
  defaultLogger = window.console;
} else if (typeof global !== "undefined") {
  defaultLogger = global.console;
}

const generateLogger = (filename) => {
  const logger = Object.create(defaultLogger);
  logger.debug = debug(`agir:${filename.replace("/", ":").replace(".js", "")}`);

  return logger;
};
```